### ISSUE TITLE
'softReset()' fails to reduce power.

### ISSUE THREAD

**Issue by Tristan-Day** [adafruit/Adafruit_SGP30#28]
- Arduino board:  ESP32 Lolin D32 Lite
- Arduino IDE version (found in Arduino -> AbouE**t Arduino menu):  N/A (PlatformIO Core, version 6.1.13)

Replication:
1. Call 'begin()' with no parameters.
2. Take a measurement.
3. Call 'softReset()' 

Outcome:
* The method 'softReset()' returns false.
* Power consumption does not change.

**Comment by Tristan-Day** [comment]
Update:

By issuing a "general reset" not using BusIO, the command functions as expected. This requires a target address of `0x00`, not the sensor address automatically added. 

A solution would be to manually pass the Wire interface as an optional argument since a pointer to the interface is not exposed by `i2c_dev`.

```cpp
boolean Adafruit_SGP30::softReset(TwoWire* theWire = &Wire) {
  theWire->beginTransmission(0x00);
  theWire->write(0x06);
  return theWire->endTransmission();
}
```

**Comment by caternuson** [comment]
Bummer it doesn't just work with the address, even though it goes against the datasheet:

![image](https://github.com/adafruit/Adafruit_SGP30/assets/8755041/2515db24-6629-4fc8-9fae-3188adf3f43c)

Sounds like you've verified it must done per datasheet.

Have you gone as far as verifying the sensor works OK again after an actual soft reset (using your modified code)?

![image](https://github.com/adafruit/Adafruit_SGP30/assets/8755041/84882e84-ac71-4683-afa3-95a042a8b888)

**Comment by Tristan-Day** [comment]
Yes, I have it restart every 5m and it seems to work fine after the first few readings.

It seems to read 400 for about 10s but after that I beleive it gives genuine readings.
